tests/testthat/test-structural.properties.R

```diff
@@ -188,15 +189,15 @@ test_that("bfs() works", {
       unreachable = FALSE,
       order = TRUE,
       rank = TRUE,
-      father = TRUE,
+      parent = TRUE,
       pred = TRUE,
       succ = TRUE,
       dist = TRUE
     )
   })
 })
 
-test_that("bfs() deprecated argument", {
+test_that("bfs() deprecated arguments", {
   g <- graph_from_literal(a - +b - +c, z - +a, d)
```

Comment:
#673 would take care of the style here. Can we somehow add this to the package (with `.Rbuildignore`) and configure/document the usage?

Reply:
https://github.com/igraph/rigraph/issues/1340